src/type/molecule.f90

```diff
@@ -488,29 +488,35 @@ end subroutine deallocate_molecule
 subroutine update(self)
    use xtb_mctc_accuracy, only : wp
    use xtb_pbc_tools
-   implicit none
-   class(TMolecule),intent(inout) :: self  !< molecular structure information
 
+   implicit none
```

Comment:
`implicit none` is already declared for the whole module and should thus be removed here.

Reply:
You are right, but always adding this statement is still good practice. It will not affect the code in any way, just some protection for the implicit typing.